src/app/contexts/UserContext/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { node } from 'prop-types';
+import { getCookiePolicy, personalisationEnabled } from './cookies';
+
+export const UserContext = React.createContext({});
+
+export const UserContextProvider = ({ children }) => {
+  const [cookiePolicy, setCookiePolicy] = useState(getCookiePolicy());
+
+  const updateCookiePolicy = () => setCookiePolicy(getCookiePolicy());
```

Comment:
I think we should move this to an action reducer patter n down the road, as we should not be directly updating the state object. Not a problem right now though.